mix.exs

```diff
@@ -28,6 +28,11 @@ defmodule Openreevoo.Mixfile do
   #
   # Type `mix help deps` for more examples and options
   defp deps do
-    []
+    [
+      # Automatic test runner
+      {:mix_test_watch, only: :dev},
```

Comment:
Do these tasks get run automatically, or is there a special command to use?

(If there's a special command, can you put it in the readme?)